custom_components/myskoda/image.py

```diff
@@ -59,6 +64,38 @@ def __init__(
         super().__init__(coordinator, vin)
 
 
+class StatusImage(MySkodaImage):
+    """A render of the current status of the vehicle."""
+
+    _attr_should_poll: bool = True
+
+    async def _fetch_url(self, url: str) -> httpx.Response | None:
```

Comment:
`_fetch_url()` is not called from anywhere.. is this maybe the problem why status image is never updated?

Reply:
_fetch_url is used in the parent class

Reply:
ah OK.. fyi I have tried to put some more logs in image.py but both `_fetch_url()` and also `image_url()` are called only once and never after

Reply:
I have added cache invalidation code, lets see if this solves it.